test/MinusTest.cpp

```diff
@@ -13,8 +13,9 @@
 
 namespace {
 auto table(size_t cols) {
+  using namespace ad_utility::memory_literals;
   ad_utility::AllocatorWithLimit<Id> alloc{
-      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1'000'000)};
+      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1_MB)};
```

Comment:
Somehwere in the test helpers we have a simple `ad_utility::testing::getAllocator()` function for these tests for the cases where we don't care about the amount of memory, use it here.

Reply:
Improving already existing tests should be a separate branch.